WordPress/Classes/Networking/PeopleRemote.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+
+class PeopleRemote: ServiceRemoteREST {
+    func getTeamFor(siteID: Int, success: People -> (), failure: ErrorType -> ()) {
+        let endpoint = "sites/\(siteID)/users"
+        let path = pathForEndpoint(endpoint, withVersion: ServiceRemoteRESTApiVersion_1_1)
+        let parameters = [
+            "number": 50,
+            "fields": "ID, nice_name, first_name, last_name, name, avatar_URL, roles, is_super_admin",
+        ]
+
+        api.GET(path,
+            parameters: parameters,
+            success: {
+                (operation, responseObject) -> Void in
+
+                if let people = try? self.peopleFromResponse(responseObject, siteID: siteID) {
+                    success(people)
+                } else {
+                    failure(Error.DecodeError)
+                }
+            },
+            failure: {
+                (operation, error) -> Void in
+                failure(error)
+        })
+    }
+
+    private func peopleFromResponse(responseObject: AnyObject, siteID: Int) throws -> People {
+        let response = responseObject as? [String: AnyObject]
+        let users = response.flatMap { return $0["users"] as? [[String: AnyObject]] }
+        guard let unwrappedUsers = users else {
+            throw Error.DecodeError
+        }
+
+        let people = unwrappedUsers.map {
+            (user: [String: AnyObject]) -> Person? in
+            guard let ID = user["ID"] as? Int else {
+                return nil
+            }
+            guard let nice_name = user["nice_name"] as? String else {
+                return nil
+            }
+            guard let name = user["name"] as? String else {
+                return nil
+            }
+
+            let first_name = user["first_name"] as? String
```

Comment:
This is a little nit picky so feel free to ignore it, but the `_` in naming doesn't feel very sSwiftish. Perhaps rename to `firstName, lastName, etc.`?